README.md

```diff
@@ -176,7 +198,8 @@ Templates can be useful to assemble your payloads from parts
 ```
 
 ### Abstract Behaviors
-Abstract Behaviors can be used to parameterize some data
+Abstract Behaviors can be used to parameterize some data.
+When an abstract behavior and a behavior extending it both have actions defined, all of them are run when the behavior matches.
```

Comment:
Can you add a paragraph of how actions are merged and ordered?

Reply:
for example, I can see that actions are deduplicated, and order by `order asc`

Reply:
```
The actions by default run in the order defined in the mock file; you can adjust this by adding an int 'order' value. The default value for 'order' is 0.
```

```
When an abstract behavior and a behavior extending it both have actions defined, all of them are run when the behavior matches.  Actions will run in order of the 'order' field; if this is the same for two actions the action defined earlier in the abstract behavior runs first, followed by actions in the concrete behavior.
```

Reply:
Also, are they deduplicated? I don't think so.  For example you may run multiple 'sleep' if you want.

Reply:
misread that `assertEqual` function, yup, no deduplication